bot/bot.py

```diff
@@ -10,7 +11,28 @@ async def on_ready():
 async def on_message(message):
     if message.author == client.user:
         return
-    if message.content.startswith('Hello'):
-        await message.channel.send('Hello!')
+    if message.content.startswith('!Horários'):
+        mensagem = message.content
+        instrução = '!Horários'
+        args = mensagem[len(instrução) + 1:].split(' ')
+
+
+        req_ping = requests.get('http://127.0.0.1:5000/ping')
+        req_schedule = requests.get('http://127.0.0.1:5000/schedule')
+        dic= req_schedule.json()
+
+        embed= discord.Embed(title="LCC Bot", descripition="Aulas do dia:")
+
+        for lcc in dic.keys():
+            if lcc.lower() in args or len(args):
+                if len(dic[lcc]) == 0:
+                    embed.add_field(name = lcc.upper(), value="Hoje não haverá aula", inline= False)
+                else:
+                    aula = dic[lcc][0]
+                    embed.add_field(name = lcc.upper(), value=("Hoje tem aula de %s ás %s horas.\n" %(aula['course'], aula['startTime'])), inline= False)
```

Comment:
Caso o número de aulas pro LCC atual, no dia, for maior que um, (como definido no exemplo de json na issue #21), esse trecho de código só irá mostrar ao usuário uma das aulas.

Acredito que iterar sobre as aulas do dia ex.: 
```python
for aula in dic[lcc]:
  # [...]
```
deve resolver o problema.